stylus/cozy-ui/defaults.styl

```diff
@@ -32,7 +32,20 @@ textarea
 
 // UTILS
 @import '../ui-base/mixins'
+@import '../ui-base/palette'
 :global
+    .coz-error
+        color  pomegranate
+
+        &.coz-error--warning:before
```

Comment:
> you want to display an error text in red then [use this class](http://tachyons.io/docs/themes/skins/). But if you want a big component, [you'll have to extend it](http://tachyons.io/components/cards/album-centered/index.html) mate, too big for the default css

Have I ever mentioned how wonderful tachyons is? Yes? What do you mean "get the fuck out of here"?